### PR TITLE
Revamp The Admin Page

### PR DESCRIPTION
**Pull request by noxymon**
dear mas @danpros,
i've done some revamp of admin page. please do some review.

### CODE REVIEW

**Comment by danpros** [comment]
@noxymon OK I will try to review it first, thanks 👍

**Comment by trendschau** [comment]
Hi @danpros and @noxymon  I am also working on a new dashboard for htmly. Not ready for a pull request yet, but please check out the screenshots in my fork: https://github.com/trendschau/tumbly

Dashboard

**Comment by fahmi182** [comment]
That's great! please pull :)

File: config/config.ini [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 ; Your timezone
 timezone = "Asia/Jakarta"
 
 ; Blog info
 blog.title = "HTMLy"
-blog.tagline = "Just another HTMLy blog"
+blog.tagline = "Let's get rocks !"
 blog.description = "Proudly powered by HTMLy, a databaseless blogging platform."
-blog.copyright = "(c) Your name."
+blog.copyright = "@noxymon"
```
Comment (Kanti) [inline config/config.ini]: please remove this change

File: config/config.ini [hunk 2]
```diff
@@ -20,7 +20,7 @@ static.frontpage = "false"
 blog.enable = "false"
 
 ; Social account
-social.twitter = "https://twitter.com"
+social.twitter = "https://twitter.com/noxymon"
```
Comment (Kanti) [inline config/config.ini]: please remove this change

File: config/config.ini [hunk 3]
```diff
@@ -34,15 +34,15 @@ blog.menu = ""
...
+comment.system = "disqus"
 
 ; Facebook comments
 fb.appid = ""
 fb.num = "5"
 fb.color = "light"
 
 ; Disqus comments
-disqus.shortname = ""
+disqus.shortname = "noxymon"
```
Comment (Kanti) [inline config/config.ini]: please remove this change

File: config/config.ini [hunk 4]
```diff
@@ -34,15 +34,15 @@ blog.menu = ""
 breadcrumb.home = "Home"
 
 ; Comment system. Choose "facebook", "disqus", or "disable".
-comment.system = "disable"
+comment.system = "disqus"
```
Comment (Kanti) [inline config/config.ini]: please remove this change

File: config/users/noxymon.ini [hunk 5]
```diff
@@ -0,0 +1,3 @@
+password = $2y$10$N1zg7R/LDbBDQwkYpVahHOnHsk15jPM3Z6HPtjDqBmJ2moqo/3FC2
```
Comment (Kanti) [inline config/users/noxymon.ini]: please remove this file

File: nbproject/project.properties [hunk 6]
```diff
@@ -0,0 +1,7 @@
+include.path=${php.global.include.path}
```
Comment (Kanti) [inline nbproject/project.properties]: please remove all Files from nbproject/*

File: nbproject/project.xml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Kanti) [inline nbproject/project.xml]: please remove all Files from nbproject/*

File: system/plugins/urlify/.gitignore [hunk 8]
```diff
@@ -0,0 +1,2 @@
+composer.phar
```
Comment (Kanti) [inline system/plugins/urlify/.gitignore]: why is the Package urlify not included via Composer?

File: .gitignore [hunk 9]
```diff
@@ -0,0 +1,7 @@
+/cache/clean.sh
+/content/data/
+/content/static/
+/content/noxymon/
+/themes/bootstrap/nbproject/private/
+/themes/bootstrap/
+/nbproject/
```
Comment (Kanti) [inline .gitignore]: please add a newline at the end of this file

File: composer.json [hunk 10]
```diff
@@ -1,25 +1,22 @@
 {
 	"config": {
 		"vendor-dir": "system/vendor/",
-		"optimize-autoloader": true,
-		"platform": {
-			"php": "5.3"
```
Comment (Kanti) [inline composer.json]: could you please tell us, why did you remove the platform php config?

File: composer.json [hunk 11]
```diff
@@ -1,25 +1,22 @@
...
     },
 	"autoload": {
 		"files": [
 			"system/includes/dispatch.php",
 			"system/includes/functions.php",
 			"system/admin/admin.php",
 			"system/includes/session.php",
-			"system/includes/opml.php"
+			"system/includes/opml.php",
+			"system/plugins/urlify/URLify.php"
```
Comment (Kanti) [inline composer.json]: Why did you added this into autoload and not as composer package?

File: .gitignore [hunk 12]
```diff
@@ -0,0 +1,7 @@
+/cache/clean.sh
+/content/data/
+/content/static/
+/content/noxymon/
+/themes/bootstrap/nbproject/private/
+/themes/bootstrap/
```
Comment (Kanti) [inline .gitignore]: could you add this theme fully? (not as submodule)
or remove it from this pull request.

**Comment by danpros** [comment]
Already creating the new one. See d1f7d27
